pom.xml

```diff
@@ -5,11 +5,11 @@
   <artifactId>editorconfig</artifactId>
```

Comment:
:+1:  for reducing it to 2 spaces, maybe we should add an `.editorconfig` file to our plugin project to define styles for XML and to be a role model. :smiley: